packages/infra/client/manager/src/utils/sql.rs

```diff
@@ -65,3 +74,51 @@ impl<'c> DerefMut for Transaction<'c> {
 		self.conn
 	}
 }
+
+/// Executes queries and explicitly handles retry errors.
+pub async fn query<'a, F, Fut, T>(mut cb: F) -> Result<T>
```

Comment:
why is this needed? we don't have crdb nor crdb retries

Reply:
This was moved, and this is for handling sql retries on the manager

Reply:
`TransactionRetryWithProtoRefreshError` is specific to cockroachdb.

the manager isn't querying crdb, right?

Reply:
It was copied over from the main runtime, no its not querying crdb